docs/content-update-details.md

```diff
@@ -30,7 +30,10 @@ The current list of namespaces excluded from the controller's watches:
 - openshift-sdn
 - openshift-service-ca-operator
 
-The list is not yet configurable, but most likely will become so as the project's lifecycle progresses.
+The list is configurable by informing `--ignorenamespace` to the `hostpath` plugin instance. The
+plugin can also be configured with `--readonly` flag, which makes the controller read the shared
```

Comment:
need to update this flag / text with whatever we decide here

Reply:
yep either refresh/reload resources ^^

Reply:
Please consider latest changes.